### PR TITLE
Add task features:base

### PR DESCRIPTION
**Pull request by vmoravec**
Task `rake features:base` will trigger tests for main components, sofar covered are these:
- admin node
- control node
- users

There are more tests to come soon.

### CODE REVIEW

**Comment by rsalevsky** [comment]
+1

File: Gemfile.lock [hunk 1]
```diff
@@ -14,7 +14,7 @@ PATH
       rspec (~> 3.2.0)
 
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.org/
```
Comment (aspiers) [inline Gemfile.lock:17]: Isn't this a regression?  bundler explicitly deprecates http here, or at least it used to.

Comment (rsalevsky) [inline Gemfile.lock:17]: This was already introduced by #8.

Comment (aspiers) [inline Gemfile.lock:17]: Then it shouldn't be in this PR.  Even if there is a good reason to include it in this PR, it should be in a separate commit with an explanation in the commit message.

Comment (aspiers) [inline Gemfile.lock:17]: I guess this is what someone meant when they put a "merciless reviewer" post-it note on my back during the CSM workshop ;-)
